qemu/tests/cfg/block_commit.cfg

```diff
@@ -0,0 +1,31 @@
+# Below case works stablly on QMP monitor, but have no more test with Human
+# monitor, so suggest to use QMP monitor as default qemu monitor run below test
+#
+- block_commit:
+    no raw qed vmdk
+    backup_image_before_testing = yes
+    restore_image_after_testing = yes
+    wait_finished = yes
+    source_image = image1
+    default_speed_image1 = 0
+    snapshot_chain = "images/sn1"
+    # If image size increase or limited committing please increase
+    # wait_timeout to avoid wait for committing finished timeout;
+    wait_timeout = 3900
+    snapshot_format = qcow2
+    kill_vm = yes
+    alive_check_cmd = dir
+    tmp_dir = /tmp
+    variants:
+        - with_stress:
+            stress_test = autotest_control
+            test_control_file = stress.control
+            # Use a low stress to make sure guest can response during stress
+            control_args = "--cpu 4 --io 4 --vm 2 --vm-bytes 256M"
+            test_timeout = 1800
+            check_cmd = 'pgrep stress'
```

Comment:
Hi @xiywang ,

Does this case support Windows guest?
If yes, I think also needs add cfg for windows.